docs/grid/custom_bulk_action.md

```diff
@@ -1,4 +1,4 @@
-Creating custom Bulk Action
+Creating custom Bulk Actions
```

Comment:
Same question about file header as in previous https://github.com/Sylius/Stack/pull/180#discussion_r1888383950